contributor-guide/modules/ROOT/pages/testing/continuous-integration.adoc

```diff
@@ -1,71 +1,94 @@
 = Continuous Integration
 
+
 The Boost project uses Continuous Integration (CI) to ensure the quality and integrity of its code. CI is the practice of merging all developers' working copies to a shared mainline several times a day. The main aim is to prevent integration problems, which are detected and can be fixed as early as possible.
```

Comment:
```suggestion
The Boost project utilizes Continuous Integration/Continuous Deployment (CI/CD) to ensure the quality and integrity of its code. CI/CD is the practice of merging all developers' working copies into a shared mainline several times a day. The primary objective is to prevent integration issues, which can be identified and addressed as soon as possible.
```